manager/allocator/network.go

```diff
@@ -991,7 +994,7 @@ func (a *Allocator) procUnallocatedNetworks(ctx context.Context) {
 		log.G(ctx).WithError(err).Error("Failed to commit allocation of unallocated networks")
 	}
 
-	for _, n := range allocatedNetworks[:committed] {
+	for _, n := range committedNetworks[:committed] {
```

Comment:
At this point `committed == len(committedNetworks)` I think, so you could just do `range committedNetworks` and possibly `assert` the equality somewhere if you were feeling defensive.